test/fast_layers.jl

```diff
@@ -38,6 +44,12 @@ fsgrad = Flux.Zygote.gradient((x,p)->sum(fs(x,p)),x,pd)
 @test fdgrad[1] ≈ fsgrad[1]
 @test fdgrad[2] ≈ fsgrad[2] rtol=1e-5
 
+fdcgrad = Flux.Zygote.gradient((x,p)->sum(fdc(x,p)),x,pd)
+@test fdgrad[1] ≈ fdcgrad[1]
+@test fdgrad[2] ≈ fdcgrad[2] rtol=1e-5

```

Comment:
why so high of a tolerance? Seems like an issue?

Reply:
would 1e-9 be ok, any specific value?

Reply:
1e-9 is fine. For something that's just changing to no allocs, I would expect it to pass at like `1e-12` at least.